guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc

```diff
@@ -74,12 +74,19 @@ ifndef::satellite[]
 +
 Otherwise the installer cannot generate the right principal name that is needed to join the realm.
```

Comment:
```suggestion
Otherwise, the installer cannot generate the right principal name that is needed to join the realm.
```

Reply:
Maybe instead of "the installer", use "`{foreman-installer}`".